script/Ciel/Day 6/06_01_CIEL06_6.txt

```diff
@@ -365,7 +365,7 @@ You prioritize your ideals over your own existence.
 [sha:d8b70243ed8fb64a6ab3116465efa190a51d2c37]{
 -- Page 247, Offset 13809.
 -- だが―――それでは命題たる永遠には程遠いぞ、蛇よ』
-However―――What you have proposed is a far cry from eternity, Serpent.'
+However――what you have proposed is a far cry from eternity, Serpent.'
```

Comment:
probably just a comma?
JP uses dash because there's two conjunctions at the start, I think